src/components/molecules/NavBar/NavBar.scss

```diff
@@ -1,6 +1,8 @@
 @import "scss/constants.scss";
 
-$border-radius: 28px;
+$menu-item-size: 32px;
+$volume-popover-width-with-sliders: 30em;
+$volume-popover-width-without-sliders: 15em;
```

Comment:
Currently no sliders are being used, so when they land, one of these constants should be deleted and the other one used.